Argo/Types/StandardTypes.swift

```diff
@@ -72,6 +72,10 @@ public extension CollectionType where Generator.Element: Decodable, Generator.El
   }
 }
 
+public func decodeArray<T: Decodable where T.DecodedType == T>(j: JSON) -> Decoded<[T]> {
```

Comment:
Variable Name Min Length Rule Violation: Variable name should be 2 characters or more: currently 1 characters (variable_name_min_length)